peewee.py

```
@@ -5135,7 +5138,8 @@ class ForeignKeyField(Field):
     def __init__(self, model, field=None, backref=None, on_delete=None,
```

Pylint (3.3.7)

[refactor] Too many arguments (14/5)

(R0913)